app/experimenter/nimbus-ui/README.md

```diff
@@ -362,7 +361,7 @@ const mkSimulatedQueries = ({
 const Subject = ({ simulatedQueries = mkSimulatedQueries() }) => {
   const mockLink = new SimulatedMockLink(simulatedQueries, false);
   return (
-    <MockedProvider link={mockLink} addTypename={false}>
```

Comment:
Oh I thought we needed this in order to drop `__typename`! TIL

Reply:
The default is `__typename` gets added by `MockedProvider`, but since we were manually defining them everywhere we added `addTypename={false}` to prevent doubling up.